framework/OpenMod.Core/Commands/CommandWrongUsageException.cs

```diff
@@ -12,7 +12,7 @@ public CommandWrongUsageException(string message) : base(message)
             
         }
 
-        public CommandWrongUsageException(ICommandContext context, IStringLocalizer localizer) : base(localizer["commands:errors:wrong_usage", new { CommandLine = context.GetCommandLine(), CommandPrefix = context.CommandPrefix + context.CommandAlias, context.CommandRegistration.Syntax }])
+        public CommandWrongUsageException(ICommandContext context, IStringLocalizer localizer) : base(localizer["commands:errors:wrong_usage", new { CommandPrefix = (context.ParentContext != null ? context.ParentContext.CommandAlias : context.CommandAlias) + context.CommandPrefix, context.CommandRegistration.Syntax }])
```

Comment:
This needs to be reverted. The context.Prefix equals to the parent context prefix + parent alias + space if a parent exists